python/twilio/README.md

```diff
@@ -0,0 +1,6 @@
+
+# Examples using Twilio
+
+This directory contains two related examples that demo integrating use of the [Twilio](https://www.twilio.com) service with the Google Cloud Vision API.
+
+[twilio-labels](twilio-labels) contains a standalone app, and [twilio-k8s](twilio-k8s) shows how you can run that app on Kubernetes to easily create a scalable service.
```

Comment:
[Optional] I'd recommend making this a bullet-point list to make it a little easier to quickly scan through.